src/Core/Checker.cpp

```diff
@@ -329,7 +327,7 @@ void Checker::check(int index, const QString &input, const QString &output, cons
     }
 }
 
-QString Checker::head(int index) const
+QString Checker::head(int index)
```

Comment:
It can be made static because this function doesn't uses any member of this class